ci/jobs/scripts/check_style/check_cpp.sh

```diff
@@ -33,6 +33,7 @@ function in_array()
 
 find $ROOT_PATH/{src,base,programs,utils} -name '*.h' -or -name '*.cpp' 2>/dev/null |
     grep -vP $EXCLUDE |
+    grep -v 'src/Storages/System/StorageSystemDashboards.cpp' |
```

Comment:
THe curly braces are problematic because of the new formatting, I guess, right?

Reply:
Yes